src/en/charms-constraints.md

```diff
@@ -179,6 +189,15 @@ model or application level):
 juju deploy apache2 --constraints "mem=4G cores=" 
 ```
 
+For the LXD cloud, we deploy PostgreSQL using a combination of an instance type
+and a specific CPU constraint. Instance 'c5.large' maps to 2 CPU and 4 GiB but
+the specific memory constraint of 3.5 GiB yields a machine with 2 CPUs and 3.5
+GiB of memory:
+
+```bash
+juju deploy postgresql --constraints "mem=3.5G instance-type=c5.large"
```

Comment:
since we're talking about things overriding I think the example would be more readable/grok-able to put the memory constraint after the instance-type so it'll read left->right as we speak of how it sorts out in your prose above.

Reply:
OK